data/mongo/mongo_test.go

```diff
@@ -0,0 +1,54 @@
+package mongo
+
+/*
+import (
```

Comment:
No tests?

Reply:
wasn't sure how to test really. But I left starter code.. Ill finish it